docs/source/reference/api-server/api-server-admin-deploy.rst

```diff
@@ -125,17 +127,28 @@ Following tabs describe how to configure credentials for different clouds on the
 
         .. code-block:: console
 
-            $ helm upgrade --install skypilot-platform skypilot/skypilot-platform \
+            $ helm upgrade --install skypilot skypilot/skypilot-nightly --devel \
             --set gcpCredentials.enabled=true \
             --set gcpCredentials.projectId=YOUR_PROJECT_ID
 
         Replace ``YOUR_PROJECT_ID`` with your actual GCP project ID.
+    
+    .. tab-item:: Other clouds
+        :sync: other-clouds-tab
+
+        You can manually configure the credentials for other clouds by `kubectl exec` into the API server pod after it is deployed and running the relevant :ref:`installation commands<installation>`.
```

Comment:
```suggestion
        You can manually configure the credentials for other clouds by `kubectl exec` into the API server pod after it is deployed and running the relevant :ref:`installation commands<installation>`. (This will not be recovered after the api service pod get restarted)
```